StoreTestWPF.Presentation/Controls/CakeDetails.xaml.cs

```diff
@@ -0,0 +1,43 @@
+﻿using StoreTestWPF.DAL.Models;
+using StoreTestWPF.ViewModel.ViewModels;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace StoreTestWPF.Presentation.Controls
+{
+    /// <summary>
+    /// Interaction logic for CakeDetails.xaml
+    /// </summary>
+    public partial class CakeDetails : UserControl
```

Comment:
CakeDetailsControl or CakeDetailsView or CakeDetailsWindow etc. We should specify type explicitly in name.

Reply:
Fixed